core/prisma/schema.dbml

```diff
@@ -281,12 +282,14 @@ Table form_memberships {
   userId String
   memberGroup member_groups
   memberGroupId String
+  pub pubs
+  pubId String
   createdAt DateTime [default: `now()`, not null]
   updatedAt DateTime [default: `now()`, not null]
 
   indexes {
-    (formId, userId) [unique]
-    (formId, memberGroupId) [unique]
+    (formId, userId, pubId) [unique]
+    (formId, memberGroupId, pubId) [unique]
```

Comment:
I _think_ we need to expand the unique constraint here, but not sure if there is a case where this might mess things up

Reply:
That seems like a pretty low priority problem to me though, since there's not really any problem with having extra form memberships. We can always write a migration later that cleans up the duplicates and adds more restrictive constraints.

Reply:
ah okay! good catch! since it's low priority, I'm in favor of punting and maybe by then we'll have upgraded our postgres to make distinct nulls easier :)